### DISCUSSION TITLE
It doesn't extract the comments of the publications

### DISCUSSION THREAD

**Discussion by SerAlberto** [kevinzg/facebook-scraper#976] [Q&A]
Could there be some mistake? a few days ago it worked fine but now the comments are no longer extracted..
'comments_full' is always equals to [] (empty array)...
Please, any answer
Thanks

**Comment by alilou2** [comment]
am having the same problem right now

**Comment by rspadar** [comment]
I also have the same problem, did you solve it?

**Reply by alilou2** [reply]
I think it's because we did a lot of request, I tried after 4 days it works , but it's only getting 30 comments for each post !
